src/bucket/BucketIndexImpl.cpp

```diff
@@ -105,7 +104,7 @@ BucketIndexImpl<IndexT>::BucketIndexImpl(BucketManager& bm,
             params.random_seed = shortHash::getShortHashInitKey();
             params.compute_optimal_parameters();
             mData.filter = std::make_unique<bloom_filter>(params);
-            estimatedIndexEntries = fileSize / mData.pageSize;
+            auto estimatedIndexEntries = fileSize / mData.pageSize;
             CLOG_DEBUG(
                 Bucket,
                 "Bloom filter initialized with params: projected element count "
```

Comment:
How about logging `estimatedIndexEntries` for debugging purposes?

Reply:
I don't think this would be particularly useful. `estimatedIndexEntries` is directly proportional to the BucketList size, which we already have a metric for. The issue with the `IndividualIndexes` was that we were using an estimate for the reserve size. This is not the case for range index we are reserving here, where we don't use an estimate but reserve exactly how much space we will need. I think this particular metric would be difficult to reason about since it would vary significantly bucket to bucket and there's not really a visualization that I think would be actionable.

I think the underlying property here that is important is `BucketIndex` memory usage. I think that would be a worthwhile metric to add, but it would be more involved than just logging `estimatedIndexEntries`. I think it's worth doing, but isn't related to this specific issue and is something we can add later.